### PR TITLE
Add index page to storybook

### PR DESCRIPTION
**Pull request by hiaux0**
## What was done
- Add index/welcome demo page
- refactor existing to accommodate for hat
image

## How to add new demo
1. In `src\resources\elements\primeDesignSystem\demos\`
2. Add your demo component 
    1. Has to have a View model (because of how `<compose>` works here)
    2. Has to be inside a folder `myDemoComponent/myDemoComponent`
3. In `demosWelcome.html` add your component
    1. `<require>` it
    2. Add a button to link to it

## Todo
- [ ] Find a way to not having to add view models

#### Before
- have to remember url to storybook elements

#### After
- index page

## Why was this done
For the CeramicService work, I wanted to have a place in the app, where I can test things.
In other places, I would have add a `playground` route, but decided to just create a temporary "demo component" in the `/storybook` route.

I believe this refactor will be beneficial for us in the future, because it gives an overview of all components.

### CODE REVIEW

File: src/resources/elements/primeDesignSystem/demos/demos.ts [hunk 1]
```diff
@@ -1,86 +1,19 @@
...
-import {Router, RouterConfiguration} from "aurelia-router";
+import {Router} from "aurelia-router";
 
 export class Demos {
 
   router: Router;
 
-  private configureRouter(config: RouterConfiguration, router: Router) {
-    const routes = [
+  private configureRouter(config, router): void {
```
Comment (paweljanicki) [inline src/resources/elements/primeDesignSystem/demos/demos.ts:8]: Do we have to get rid of typing the arguments?

**Review by paweljanicki — APPROVED** [review]

**Comment by paweljanicki** [comment]
Great work! I like the idea, easier to use the UI and also the code is cleaner

**Comment by dkent600** [comment]
Design system has no dependency on ceramicservice, so I don't understand the need.

**Comment by hiaux0** [comment]
> Design system has no dependency on ceramicservice, so I don't understand the need.

I mentioned this in the initial comment

> For the CeramicService work, I wanted to have a place in the app, where I can test things.
In other places, I would have add a playground route, but decided to just create a temporary "demo component" in the /storybook route.

I wanted to create a "dummy app" with some "Create" and "Load" buttons calling the ceramic service

File: src/resources/elements/primeDesignSystem/demos/demos.ts (hunk 1 above)
Comment (hiaux0) [reply src/resources/elements/primeDesignSystem/demos/demos.ts:8]: Right, was lost from copy paste.

**Review by dkent600 — CHANGES_REQUESTED** [review]
Can we use the existing demos.html file, and the aurelia support for enumerating "nav" routes (see here:  https://aurelia.io/docs/routing/configuration#basic-configuration)

**Comment by hiaux0** [comment]
Keep enumaration approach for simplicty

**Pull request closed without merging**